pallets/transaction-payment/src/lib.rs

```diff
@@ -60,12 +59,12 @@ use sp_runtime::{
         InvalidTransaction, TransactionPriority, TransactionValidity, TransactionValidityError,
         ValidTransaction,
     },
-    FixedI128, FixedPointNumber, FixedPointOperand, Perquintill,
+    FixedPointNumber, FixedPointOperand, FixedU128, Perquintill,
 };
 use sp_std::prelude::*;
 
 /// Fee multiplier.
-pub type Multiplier = FixedI128;
+pub type Multiplier = FixedU128;
```

Comment:
This is used in the only storage item; should we write a migration?

Reply:
Probably best to do that. Just to be safe. ps We don't actually use the fee multiplier. It should be set as a constant 1.

Reply:
We decided it was not necessary on a meeting.